cluster/gce/gci/master-helper.sh

```diff
@@ -157,6 +157,7 @@ function create-master-instance-internal() {
   metadata="${metadata},gci-docker-version=${KUBE_TEMP}/gci-docker-version.txt"
   metadata="${metadata},kube-master-certs=${KUBE_TEMP}/kube-master-certs.yaml"
   metadata="${metadata},cluster-location=${KUBE_TEMP}/cluster-location.txt"
+  metadata="${metadata},startup-script=${KUBE_ROOT}/cluster/gce/gci/startup-script.sh"
```

Comment:
If we add a startup-script metadata, it will be executed on VM boot, right? In that case, we don't really need the systemd service, right?

Anyway, we don't want to use startup-script, because we want to execute it only conditionally, when internal-ip-address is attached. There are ways of setting up a cluster, e.g. a public ip cluster, where this internal-ip-address is not created.